src/Umbraco.Tests/PublishedContent/StronglyTypedModels/Home.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
```

Comment:
Did you check this file in by accident?

Reply:
No, this was deliberate to use with the Unit Test for this item.

Reply:
Ah yes, I see it now, cool!